core/domain/change_domain.py

```diff
@@ -23,10 +23,14 @@
 from core import feconf
 from core import utils
 
-from typing import Dict
+from typing import Dict, List, cast
 
 
-def validate_cmd(cmd_name, valid_cmd_attribute_specs, actual_cmd_attributes):
+def validate_cmd(
```

Comment:
If we remove `.get()` from here then the backend test fails. Because currently, there are some places in the codebase where these `allowed_values` and `deprecated_values` keys are not added to the `ALLOWED_COMMANDS` dict yet.

Reply:
Makes sense